benchmarks/pytorch-lightning/voirfile.py

```diff
@@ -0,0 +1,35 @@
+# Import this to instrument the ArgumentParser, remove if no such thing
+from voir.overseer import Overseer
+
+from milabench.opt import instrument_argparse
```

Comment:
Since you are using giving instead of the probes, keep only this line and remove the rest.

Reply:
Fixed in e85a2f9b278000cae304ff2437c195a169159261